ajax/libs/sticky-js/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "sticky-js",
+  "filename": "sticky.min.js",
+  "description": "Sticky-js is a library for sticky elements written in vanilla javascript. With this library you can easily set sticky elements on your website. It's also responsive.",
+  "license": "MIT",
+  "author": "rgalus <biuro@rafalgalus.pl>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/rgalus/sticky-js.git"
+  },
+  "keywords": [
+    "sticky",
+    "position",
+    "scroll",
+    "javascript"
+  ],
+  "npmName": "sticky-js",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/!(*.gz)"
+      ]
+    }
+  ]
+}
```

Comment:
There should be a newline char here, but if there is a problem to add it, then never mind.